db/discord_db/message.py

```diff
@@ -0,0 +1,14 @@
+from base import Base
+
```

Comment:
Move all modules to a module.py file instead of a file for each module.
File for each class relevant only for Python functionality.